server/src/frontend_api.rs

```diff
@@ -214,7 +214,7 @@ security(
 )
 )]
 #[get("")]
-#[instrument(skip(engine_cache, token_cache))]
+#[instrument(skip(edge_token, req, engine_cache, token_cache))]
```

Comment:
Just so I'm clear: does this man we're not logging out the token and the request anymore? Any reason we changed it? (I'm sure it's fine, just curious)

Reply:
We don't want to log the token. And the request when looking at it didn't give us enough information.

Reply:
Got it; thanks!